src/store/modules/routingSlip.ts

```diff
@@ -144,24 +144,27 @@ export default class RoutingSlipModule extends VuexModule {
   }
 
   @Action({ rawError: true })
-  public async checkRoutingNumber (): Promise<boolean> {
+  public async checkRoutingNumber (): Promise<any> {
     const context: any = this.context
     try {
-      const routingumber = context.state.routingSlipDetails.number
-      const response = await RoutingSlipService.getRoutingSlip(routingumber)
+      const routingNumber = context.state.routingSlipDetails.number
+      const response = await RoutingSlipService.getRoutingSlip(routingNumber)
       // if routing number existing we will get 200 as response
       // else we will get 204
       if (response.status === 204) {
-        // we will return truw,so can use this routing number
-        return true
+        return ''
       }
       // all other case routing is existing so can't use this number
-      return false
+      return 'exists'
```

Comment:
Not sure if it's worth creating a enum for this.

Reply:
better to use enum or const